plugins/flytekit-aws-sagemaker/setup.py

```diff
@@ -27,11 +28,12 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
Does this plugin work with Python 3.12?

Reply:
should work. updated the setup.py file. let me know if that works.